Sources/Workspace/Workspace.swift

```diff
@@ -1354,6 +1351,31 @@ extension Workspace {
         )
     }
 
+    private func loadManifests(forURLs urls: [String], diagnostics: DiagnosticsEngine) -> [Manifest] {
+        // this is allot of boilerplate code but its is important for performance
+        let operationQueue = OperationQueue()
+        operationQueue.name = self.queue.label + "-root-manifest-loading"
+        operationQueue.maxConcurrentOperationCount = (try? ProcessEnv.vars["ROOT_MANIFEST_LOADING_MAX_CONCURRENCY"].map(Int.init)) ?? 100
```

Comment:
I do think this limit is a good idea.  Would setting it to 1 be the way to disable the parallelism?

Naming nit:  From what I’ve seen, SwiftPM environment variables are usually prefixed with “SWIFTPM_”, but I could also believe if that’s not being followed consistently.  I realize that’s totally a nitpick.

I only mention it in this PR because if we start setting this (for testing or workaround purposes), and then fix the name later, then that’s more hassle.

One of the things I’d want to document at some point is the list of environment variables and what they mean, but that’s of course for later.

Reply:
>  Would setting it to 1 be the way to disable the parallelism?

yes

Reply:
> Naming nit: From what I’ve seen, SwiftPM environment variables are usually prefixed with “SWIFTPM_”, but I could also believe if that’s not being followed consistently. I realize that’s totally a nitpick.

great point. renamed